src/Compilers/VisualBasic/Portable/Scanner/CharacterInfo.vb

```diff
@@ -488,4 +488,38 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
     End Class
 
+    Friend Module CharExts
+
+        <Runtime.CompilerServices.Extension>
+        Friend Function IsFrom(c As Char, c0 As Char, c1 As Char) As Boolean
+            Return (c = c0) OrElse (c = c1)
+        End Function
+
+        <Runtime.CompilerServices.Extension>
+        Friend Function IsFrom(c As Char, c0 As Char, c1 As Char, c2 As Char) As Boolean
+            Return (c = c0) OrElse (c = c1) OrElse (c = c2)
+        End Function
+
+        <Runtime.CompilerServices.Extension>
+        Friend Function IsFrom(c As Char, c0 As Char, c1 As Char, c2 As Char, c3 As Char) As Boolean
+            Return (c = c0) OrElse (c = c1) OrElse (c = c2) OrElse (c = c3)
+        End Function
+
+        <Runtime.CompilerServices.Extension>
+        Friend Function IsntFrom(c As Char, c0 As Char, c1 As Char) As Boolean
```

Comment:
Prefer `IsNotFrom` over `IsntFrom`.  In general we don't use contractions in the APIs.